modules/microshift-nodeport-unreachable-workaround.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-known-issues.adoc
+:_content-type: PROCEDURE
+[id="microshift-nodeport-unreachable-workaround_{context}"]
+= Manually restarting the `ovnkube-master` pod to resume node port traffic
+
+There is an issue with new installs of {product-title} in which the `NodePort` service traffic might be stopped. To troubleshoot this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.
+
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* Access to the cluster as a user with the `cluster-admin` role.
```

Comment:
MicroShift doesn't support users, other than the built in admin. So, there is nothing for the user to actually do here, other than be able to access the cluster using the default kubeconfig file (i.e. set KUBECONFIG), that's generated by MicroShift and available on the host at /var/lib/microshift/resources/kubeadmin/kubeconfig